src/components/expanded-state/custom-gas/FeesPanel.tsx

```diff
@@ -11,7 +11,6 @@ import {
   Keyboard,
   KeyboardAvoidingView,
 } from 'react-native';
-// @ts-expect-error
```

Comment:
This is just to get CI passing. It won't be a diff once https://github.com/rainbow-me/rainbow/pull/4157 is merged.